tilelang/__init__.py

```
@@ -106,10 +112,48 @@ def emit(self, record):
 
+def _prepend_windows_runtime_paths():
+    if not sys.platform.startswith("win32"):
+        return
+
+    runtime_paths = []
+
+    try:
+        from tvm_ffi import libinfo as tvm_ffi_libinfo
+
+        runtime_paths.append(os.path.dirname(tvm_ffi_libinfo.find_libtvm_ffi()))
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)